test/cypress/integration/l10n.js

```diff
@@ -0,0 +1,61 @@
+// Note up to date there is no translation for any string in 'es' and 'nl'.
+const availableLanguages = ['en', 'fr', 'ja', 'zh'];
+
+const languageCheckHelper = (url, selector) => (language, message) => {
+  cy.visit(url, {
+    onBeforeLoad(win) {
+      if (language) {
+        Object.defineProperty(win.navigator, 'languages', {
+          value: [language],
+        });
+      }
+    },
+  });
+
+  cy.get(selector).should('contain.text', message);
+};
+
+describe('Localization tests with the t`String` format', () => {
+  const helper = languageCheckHelper('/ui/repositories', 'h1');
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  const translations = [
+    'Repo Management',
+    'Gestion de référentiel',
+    'リポジトリー管理',
+    '仓库管理',
+  ];
```

Comment:
```suggestion
  const translations = {
    en: 'Repo Management',
    fr: 'Gestion de référentiel',
    ja: 'リポジトリー管理',
    zh: '仓库管理',
  };
```

so that we don't have to synchronize the list of languages between 3 arrays in different parts of the file?

(and `helper(key, translations[key])` is probably a bit clearer too)

Reply:
I think in tests, especially this straightforward one it does not really matter, but if you want it its a 1min update. Then I will remove the top level language list too as it will be "duplicate"